src/SprykerSdk/Integrator/IntegratorConfig.php

```diff
@@ -111,7 +132,21 @@ public function getSharedConfigVariableName(): string
      *
      * @return string
      */
-    public function getSharedConfigPath(): string
+    public function getConfigPath(): string
```

Comment:
also please check methods visibility. I guess there is no reason to keep this method public

Reply:
this method used in ConfigureEnvManifestStrategy